mods/d2k/rules/structures.yaml

```diff
@@ -272,7 +272,6 @@ refinery:
 			TopLeft: -512, -1024
 			BottomRight: 1536, 0
 	HitShape@BOTTOM:
-		UseOccupiedCellsOffsets: false
```

Comment:
Note to reviewers:  When originally setting this up, I simply forgot that 'false' is actually the internal default, and therefore this line is redundant for all shapes that are not inherited (i.e. all `@FOO` shapes). Applies to all 3 shipping mods.